### PR TITLE
E_CLASSROOM-236 [Student][Learnings][FE/BE] Display all quizzes

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-236
### Definition of Done
- [x] All quizzes are displayed
- [x] Pagination is working, maximum of 12 per page
- [x] Display the category and subcategory where the quiz is associated
- [x] No need for filter and header information
- [x] When clicking the item, redirect to results page

### Related PR:
- https://github.com/framgia/sph-classroom-els-be/pull/47

#### Main note
- http://localhost:3003/learnings
#### Pages Affected
- N/A

### Scenarios/ Test Cases
- [ ] All quizzes are displayed
- [ ] Pagination is working, maximum of 12 per page
- [ ] Display the category and subcategory where the quiz is associated
- [ ] When clicking the item, redirect to results page
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
learnquiz1

### CODE REVIEW

**Comment by catampongan** [comment]
## Working
- [x] Displays the category and subcategory where the quiz is associated

## Comment
- [x] Scrollbar should only be available when it is scrollable and fix the position of scrollbar
- [x] Other categories that I have learned does not display on the learning page
- [x] Pagination is not working

-----
`Scrollbar`

Screenshot 2022-02-08 161804

`Categories Learned`

Screenshot 2022-02-08 161815

----------
- Total Test Time: 1.25 hr

**Comment by catampongan** [comment]
## Working
- [x] Displays the category and subcategory where the quiz is associated
- [x] Scrollbar should only be available when it is scrollable and fix the position of scrollbar
- [x] Other categories that I have learned does not display on the learning page
## Comment
- [x] Pagination is not working
---------
learnings
--------
- Total Test Time: 1.5 hr

**Comment by catampongan** [comment]
## Approved PR #94 
- [x] Displays the category and subcategory where the quiz is associated
- [x] Scrollbar should only be available when it is scrollable and fix the position of scrollbar
- [x] Other categories that I have learned does not display on the learning page
- [x] Pagination is not working
---------
Screenshot 2022-02-09 172959
Screenshot 2022-02-09 173258

--------
- Total Test Time: 1.75 hr
- Total Iterations: 2
- Total Comments: 4

File: src/pages/student/categories/CategoryList/index.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ function CategoryList() {
       setPerPage(data.per_page);
       setTotalItems(data.total);
       setLastPage(data.last_page);
+      console.log(data.data);
```
Comment (karlolee50) [inline src/pages/student/categories/CategoryList/index.js]: remove if not needed

File: src/api/Learning/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      params: {
+        ...params
+      }
+    };
+
+    return API.request(options);
+  }
+};
+
+export default LearningApi;
```
Comment (karlolee50) [inline src/api/Learning/index.js]: add extra line
```suggestion
export default LearningApi;

```

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**